.github/configs/stateless-e2e.yml

```diff
@@ -0,0 +1,57 @@
+polygon_pos_package:
+  network_params:
+    el_block_interval_seconds: 1
+
+  participants:
+    # 8 validators on Bor's stateless_sync branch
+    - kind: validator
+      cl_type: heimdall-v2
+      cl_image: heimdall-v2:stateless_sync
+      el_type: bor
+      el_image: bor:stateless_sync
```

Comment:
Since this e2e workflow could be used as a foundation for all types of e2e testing, the image could be renamed to `bor:local`, which is more generic. Also, we can rename this yml file to `kurtosis-e2e.yml`.